//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcd_NO_P.h
//
// Code generation for function 'abcd_NO_P'
//

#ifndef ABCD_NO_P_H
#define ABCD_NO_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcd_NO_P(double P_init, double V_init, double A_init, double P_wayp,
               double V_max, double A_min, double J_max, double J_min,
               double t[7]);

#endif
// End of code generation (abcd_NO_P.h)
